package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "serverless-vpc-plugin",
-  "version": "1.0.6",
+  "name": "@compassion-technology/serverless-vpc-plugin",
+  "version": "1.0.7.beta1",
```

Comment:
dont forget to change

Reply:
I'm not even sure if this will work or not so I'll probably leave it as beta until we have a `2.0.0` or `1.1.0` release with all the changes we want